src/Transit.js

```diff
@@ -18,9 +18,17 @@ function Transit() {
   
       for (let i = 0; i < json.departures.length; i++) {
         if (json.departures[i].actual) {
-          realTimeDepartures.push(json.departures[i])
+          if (realTimeDepartures.length < 6) {

```

Comment:
I think it could be a little more readable to put this condition as part of the loop check